### ISSUE TITLE
Design and support various data sources.

### ISSUE THREAD

**Issue by mstao** [mstao/asynchronous-logger-component#1]
1. File 
2. MySQL
3. Http (POST)
4. TCP / UDP

**Comment by ramasCoder** [comment]
Can I contribute to this ?

**Comment by mstao** [comment]
> Can I contribute to this ?

Of course, Now I want to design to an asynchronous logger component, which can collect data from file, MySQL and etc asynchronously, and export data to somewhere like logstash, redis and etc.  If you have any good ideas and suggestion,  please open issues or pull request. Thanks for contributing!
